plenum/test/txn_author_agreement/acceptance/test_taa_acceptance_validation.py

```diff
@@ -140,21 +148,22 @@ def test_taa_acceptance_time_near_upper_threshold(
     request_dict, latest_taa, monkeypatch
 ):
     taa_ts = latest_taa.txn_time
-    pp_time = (
-        max_last_accepted_pre_prepare_time +
-        randint(0, tconf.TXN_AUTHOR_AGREEMENT_ACCEPTANCE_TIME_AFTER_PP_TIME)
-    )
+    pp_time = max_last_accepted_pre_prepare_time
```

Comment:
Why did we remove adding the delta here?

Reply:
Because it is the addition of randomness to the test as a small time shift. Now we use the date and this is impossible.